apps/passport-server/src/util/telegramWebApp.ts

```diff
@@ -0,0 +1,16 @@
+export const htmlContent = `
```

Comment:
can we rename this to "emptyHtml" or "closeWebviewHtml" to indicate its function?

Reply:
Closed by 5c7e4e07